### PR TITLE
CHAOS-1800: adds draft chaos 0.13.x relnotes

### PR DESCRIPTION
**Pull request by StrawberryPad**
[**Link to preview**](https://647f11ffb45fb610a38406fc--harness-developer.netlify.app/release-notes/chaos-engineering)

### CODE REVIEW

File: release-notes/chaos-engineering.md [hunk 1]
```diff
@@ -15,39 +15,43 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
 :::
 
-## Latest - May 23, 2023, version 0.12.1
+## Latest - June 5, 2023, version 0.13.XXXXXXX
```
Comment (Jonsy13) [inline release-notes/chaos-engineering.md:18]:
```suggestion
## Latest - June 5, 2023, version 0.13.3
```

File: release-notes/chaos-engineering.md [hunk 2]
```diff
@@ -15,39 +15,43 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* Previously, the Last Heartbeat value was empty when chaos infrastructures were pending. Now, to prevent user confusion, this value displays N/A when chaos infrastructures are pending. (CHAOS-1666)
+
+* Enhanced Chaos Infrastructures table to allow routing to corresponding connectors from the Chaos Infrastructures screen. (CHAOS-1665)
+
+* When scheduling an experiment fails for any reason, the user now sees the error when hovering over the status. (CHAOS-1574)
+
+* Added a new advanced configuration to allow users to add annotations to all chaos pods using the UI. (CHAOS-1465) 
+
+* Upgraded argo-workflows library to version 3.2.11. This fixes vulnerabilities found in some of our components, such as chaos-manager and subscriber. (CHAOS-568)
```
Comment (Jonsy13) [inline release-notes/chaos-engineering.md:39]: This one we can remove

File: release-notes/chaos-engineering.md [hunk 3]
```diff
@@ -15,39 +15,53 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+## Latest - June 5, 2023, version 0.13.4
 
 ```mdx-code-block
 <Tabs>
   <TabItem value="What's new">
 ```
 
-* Reports can now be downloaded. (CHAOS-1615)
+:::caution
+This release breaks backward compatibility with older chaos infrastructures. You must update Chaos Infrastructures and the chaosnative/go-runner image in experiment definitions. If you don't upgrade, then chaos experiments will start to fail.
```
Comment (roshnisarangadharan) [inline release-notes/chaos-engineering.md:26]: Chaos Infrastructure or chaos infrastructures ?

Comment (roshnisarangadharan) [reply release-notes/chaos-engineering.md:26]: global comment

File: release-notes/chaos-engineering.md [hunk 4]
```diff
@@ -15,39 +15,53 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* Added audit events for various GameDay operations such as create, update, etc., so that users can easily audit operations done on their GameDays. (CHAOS-1709)
+
+* Browser tabs now show the module page name to help users switching between different tabs. (CHAOS-1683)
+
+* The Delete Chaos Infrastructure API has been updated to allow deletion of only one infrastructure. (CHAOS-1681)
+
+* Previously, the Last Heartbeat value was empty when chaos infrastructures were pending. Now, to prevent user confusion, this value displays N/A when chaos infrastructures are pending. (CHAOS-1666)
+
+* Enhanced Chaos Infrastructures table to allow routing to corresponding connectors from the Chaos Infrastructures screen. (CHAOS-1665)
```
Comment (roshnisarangadharan) [inline release-notes/chaos-engineering.md:45]:
```suggestion
* Enhanced the Chaos Infrastructures table to allow routing to corresponding connectors from the Chaos Infrastructures screen. (CHAOS-1665)
```

File: release-notes/chaos-engineering.md [hunk 5]
```diff
@@ -15,39 +15,53 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
   </TabItem>
   <TabItem value="Fixed issues">
 ```
 
-* Corrected the UI text for the Inactive and Pending states for Linux infrastructure states. (CHAOS-1633)
-
-* Improved the UI text when there are empty search results for Kubernetes or Linux infrastructures. (CHAOS-1629)
-
-* Corrected the UI text for Linux infrastructure screens. (CHAOS-1619) 
+* Improved the UI message returned when user searches for a GameDay and the search term is not found. Now the message more accurately states "No GameDay found matching the search term." (CHAOS-1717)
```
Comment (roshnisarangadharan) [inline release-notes/chaos-engineering.md:62]:
```suggestion
* Improved the UI message when user searches for a GameDay and the search term is not found. Now the message more accurately states "No GameDay found matching the search term." (CHAOS-1717)
```

**Review by roshnisarangadharan — APPROVED** [review]

**Review by Jonsy13 — DISMISSED** [review]

**Pull request merged**